public-api/src/main/java/org/pmiops/workbench/publicapi/DataBrowserController.java

```diff
@@ -194,7 +239,47 @@
 
     @Override
     public ResponseEntity<DbDomainListResponse> getDomainSearchResults(String keyword){
-        List<DbDomain> domains=dbDomainDao.findDomainSearchResults(keyword);
+
+        keyword = ConceptService.modifyMultipleMatchKeyword(keyword);
+        List<DbDomain> domains = dbDomainDao.findDomainSearchResults(keyword);
+        DbDomainListResponse resp = new DbDomainListResponse();
+        resp.setItems(domains.stream().map(TO_CLIENT_DBDOMAIN).collect(Collectors.toList()));
+        return ResponseEntity.ok(resp);
+    }
+
+    @Override
+    public ResponseEntity<ConceptListResponse> getAdvancedConceptSearch(SearchConceptsRequest searchConceptsRequest){
+
+        Integer maxResults=searchConceptsRequest.getMaxResults();
+        if(maxResults == null){
+            maxResults=25;
+        }
+        StandardConceptFilter standardConceptFilter=searchConceptsRequest.getStandardConceptFilter();
+        if(standardConceptFilter == null){
+            standardConceptFilter = StandardConceptFilter.ALL_CONCEPTS;
+        }else if("S".equals(standardConceptFilter)){
```

Comment:
This logic doesn't make sense anymore. You are getting a StandardConceptFilter in the request... all you should have to do is set it to ALL_CONCEPTS if it's null.

Reply:
Makes sense. Thanks.